{
  try {
    maybeThrow(x);
    return 1;
  }
 catch (  RuntimeException ex) {
    return 2;
  }
}
